CHANGELOG.md

```diff
@@ -200,6 +223,9 @@ migrate tests from prior versions.
 [0.8.1]: https://github.com/dogmatiq/testkit/releases/tag/v0.8.1
 [0.9.0]: https://github.com/dogmatiq/testkit/releases/tag/v0.9.0
 [0.10.0]: https://github.com/dogmatiq/testkit/releases/tag/v0.10.0
+[0.11.0]: https://github.com/dogmatiq/testkit/releases/tag/v0.11.0
+
+[v0.11.0 migration guide]: https://github.com/dogmatiq/testkit/blob/v0.11.0/docs/MIGRATING-v0.11.0.md
```

Comment:
Note that this URL refers to the v0.11.0 tag explicitly, which doesn't exist yet.  I've done this so this link will work even if we move these migration docs elsewhere in some future tag.